mock/private/check.rkt

```diff
@@ -11,8 +11,13 @@ provide
     check-mock-num-calls (-> exact-nonnegative-integer? mock? void?)
 
 
-(define-simple-check (check-mock-called-with? args mock)
-  (mock-called-with? args mock))
+(define-check (check-mock-called-with? args mock)
+  (let ([result (mock-called-with? args mock)])
+    (with-check-info
+      [('Expected args) ('Actual (mock-calls mock))]
```

Comment:
Nit: to match the format of other check info, `expected` and `actual` should be lowercase